gtc_to_vcf.py

```diff
@@ -286,13 +286,21 @@ def main():
     parser.add_argument("--auxiliary-loci", dest="auxiliary_loci", default=None, required=False, help="VCF file with auxiliary definitions of loci (optional)")
     parser.add_argument("--filter-loci", dest="filter_loci", default=None, required=False, help="File containing list of loci names to filter from input manifest (optional)")
     parser.add_argument("--disable-genome-cache", dest="disable_genome_cache", default=False, action="store_true", help="Disable caching of genome reference data")
+    parser.add_argument("--include-attributes", dest="include_attributes", default=[], choices=["BAF","LRR"], nargs="*", help="Additional attributes to include in VCF FORMAT output (optional)")
```

Comment:
Can the choices here be set from a static list in the FormatFactory that it claims to know how to generate Formats for?